python/src/aiconfig/default_parsers/hf.py

```diff
@@ -120,7 +120,7 @@ class HuggingFaceTextGenerationParser(ParameterizedModelParser):
     A model parser for HuggingFace text generation models.
     """
 
-    def __init__(self, model_id: str = None, use_api_token=False):
+    def __init__(self, model_id: str = None, use_api_token=True):
```

Comment:
Why was this false before and why is it set to True now?

Reply:
It's because after using it for a bit, you end up getting rate-limited, so this needs to be true so that you can by default fetch your API key